docs/mpz.rst

```diff
@@ -105,6 +105,7 @@ mpz Functions
 .. autofunction:: powmod_exp_list
 .. autofunction:: powmod_base_list
 .. autofunction:: powmod_sec
+.. autofunction:: prev_prime
```

Comment:
Not sure if we should include this function to the sphinx docs.

But if so, you should do this conditionally, probably with the ["only" directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-only) of the sphinx.  Also, I think that in this case we should mention in the prev_prime docstring, that this function is available only for certain conditions.

Reply:
```suggestion
.. function:: prev_prime(x, /) -> mpz

   Return the previous *probable* prime number < x.
   Only present when compiled with GMP 6.3.0 or later.

```

Reply:
It took me a few days but I made a conditional tag and got this working with the only directive (thanks for pointing me at it).

Reply:
The problem with your solution is that this function will not be displayed on the https://gmpy2.readthedocs.io/en/latest/ for a long time.  That's why I did this suggestion.

Reply:
Yep you were persepctive in a way I had to learn by making the mistake. Should be fixed now. I tried to run build, tests, and sphinx, with and without gmp 6.3.0.